src/Aquifer.API/Endpoints/Resources/Content/SendForPublisherReview/Community/Endpoint.cs

```diff
@@ -0,0 +1,60 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.Content.SendForPublisherReview.Community;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService, IResourceHistoryService historyService) 
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Post("/resources/content/{ContentId}/send-for-publisher-review-community");
+        Permissions(PermissionName.SendReviewCommunityContent);
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var contentVersion = await dbContext.ResourceContentVersions
+            .Where(x => x.ResourceContentId == request.ContentId && x.AssignedUserId == user.Id && x.IsDraft)
+            .Include(x => x.ResourceContent)
+            .SingleAsync(ct);
```

Comment:
This will blow up if nothing is found. `Single` means there must be 1 and only 1 or it will blow up. `First` means there must be at least 1 and it will return the first 1, or it will blow up. `SingleOrDefault` and `FirstOrDefault` will allow a case where nothing is found. So we want `SingleOrDefaultAsync` here to say that we allow either no record or 1 and only 1 record.